src/features/universities/university-route-validation.ts

```diff
@@ -0,0 +1,22 @@
+import * as Joi from '@hapi/joi';
+
+export const getUniversity = {
+    params: {},
+    query: {
+        limit: Joi.number().min(1).max(20).default(10),
+        offset: Joi.number().default(0),
+    },
+    body: {},
+};
+
+export const postUniversity = {
+    params: {},
+    query: {},
+    body: {
+        universityName: Joi.string().required(),
+        profEmailDomain: Joi.string().required(),
+        studentEmailDomain: Joi.string().required(),
+        verifyInstitutionalEmail: Joi.bool().optional().default(true),
+        paidUntil: Joi.date().optional().default(new Date()),
```

Comment:
I don't think this does what you think it does,

This will default paidUntil to server startup date

Probably fine either way

Reply:
Good catch, I didn't think about that. It is probably still fine.